website/integrations/services/gatus/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Gatus
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Gatus?
+
+> Gatus is a free and open source project for endpoint monitoring. It allows many types of monitoring from pings or http requests to DNS checking and certification expiration. This is all done through yaml files.
+>
+> -- https://github.com/TwiN/gatus
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `gatus.company` is the FQDN of the Gatus install.
+-   `authentik.company` is the FQDN of the authentik install.
+
+## Gatus configuration
+
+1. Within the main configuration file of Gatus add this to it:
+
+```yml
+security:
+    oidc:
+        issuer-url: https://authentik.company/application/o/gatus/
+        client-id: "CLIENT_ID"
+        client-secret: "CLIENT_SECRET"
+        redirect-url: https://gatus.company/authorization-code/callback
+        scopes: [openid]
+```
+
+:::note
+Gatus auto updates the configuration about every 30 seconds but it does not pick up the changes just restart the instance
+:::
+
+## authentik configuration
```

Comment:
Maybe put the authentik config section first, so that client id and secret are available when configuring gatus.